File: src/main/java/ru/ioleksiv/telegram/bot/api/model/annotation/validator/impl/UserFilterValidator.java
package ru.ioleksiv.telegram.bot.api.model.annotation.validator.impl;

import ru.ioleksiv.telegram.bot.api.model.annotation.validator.CustomValidator;
import ru.ioleksiv.telegram.bot.api.model.annotation.validator.FilterValidator;
import ru.ioleksiv.telegram.bot.api.model.objects.std.User;

public class UserFilterValidator extends FilterValidator<User> {

    public UserFilterValidator(CustomValidator<User> validator) {
        super(User.class, validator);
    }
}

File: core/src/main/kotlin/io/github/oleksivio/tl/kbot/core/model/annotation/validator/impl/UserFilterValidator.kt
package io.github.oleksivio.tl.kbot.core.model.annotation.validator.impl

import io.github.oleksivio.tl.kbot.core.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.std.User
import io.github.oleksivio.tl.kbot.core.core.controller.handler.check.Validator

class UserFilterValidator(validator: Validator<User>) : FilterValidator<User>(
    User::class, validator)
